src/Kliva/Models/ActivitySummaryService.cs

```diff
@@ -0,0 +1,83 @@
+using System;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using DynamicData;
+using DynamicData.Binding;
+using GalaSoft.MvvmLight.Threading;
+using Kliva.Services.Interfaces;
+
+namespace Kliva.Models
+{
+    public class ActivitySummaryService : IDisposable
+    {        
```

Comment:
Purely based on the given name, I'm no fan of the placement inside the models folder... or better describe the class or move to services folder.
But seeing this is more a class to feed data to UI, I'm more inclined to call is 'factory' or something like that?
@bartlannoeye any suggestions?